src/main/scala/com/fulcrumgenomics/commons/io/PathUtil.scala

```diff
@@ -28,23 +28,29 @@ import java.nio.file.{Path, Paths}
 
 /** Provides utility methods for creating and manipulating Path objects and path-like Strings. */
 object PathUtil {
-  val illegalCharacters: String = "[!\"#$%&'()*/:;<=>?@\\^`{|}~] "
+  val MaxFileNameSize: Int = 254
+  val IllegalCharacters: String = "[!\"#$%&'()*/:;<=>?@\\^`{|}~] "
```

Comment:
Do you want to have a method `def illegalCharacters: String = IllegalCharacters` that's deprecated so we don't break the API?